client/src/Components/Input/Input.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { InputComponent } from './InputStyles';
+
+export type InputProps = {
```

Comment:
Nie mieliśmy używać interface?

Reply:
Pierwsze słyszę.. :p dostosowałem się do aktualnego zapisu, także jak interfejsy to już poprawiam :)